content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,113 @@
+
+In libp2p:
+
+- WebRTC multiaddresses are composed of a standard UDP multiaddr,
+  followed by `webrtc` and the `multihash` of the certificate that
+  the node uses, as such:
+  `/ip4/1.2.3.4/udp/1234/webrtc/certhash/<hash>/p2p/<peer-id>`;
+- WebRTC offers security via DTLS. Peers still need to authenticate remote peers
+  by their libp2p identity.
+
+A browser can connect to a server node without needing a trusted TLS
+certificate. View [this scenario](https://github.com/libp2p/specs/blob/master/webrtc/README.md#browser-to-public-server) on browser-to-server connection establishment
+in the technical specification.
+
+In general, a server will act as an [ICE Lite](https://www.rfc-editor.org/rfc/rfc5245)
+agent and binds to a UDP port waiting for incoming STUN and SCTP packets and multiplexes
+based on source IP and source port.
+
+When a browser wants to establish a WebRTC connection to a server, it instantiates
+an `RTCPeerConnection`. The browser constructs the server's SDP answer locally
+based on the browser's multiaddr. The browser creates a local offer via
+`RTCPeerConnection.createOffer()`. A sets the same username and password on the
+local offer as in the remote SDP answer.
+
+{{< alert icon="" context="info" text="SDP (Session Description Protocol) is a protocol that is used to describe multimedia sessions. The SDP answer is a message sent by a server in response to an SDP offer from a client. The offer and answer are used to establish a session between a client and server, allowing them to exchange media." />}}
+
+Once the browser sets the SDP offer and answer, it will send STUN requests to
+the server. The browser and server then execute the DTLS handshake as part of the
+standard WebRTC connection establishment. DTLS is similar to TLS but runs on an
+unreliable transport instead of on top of an ordered byte stream (like TCP).
+
+A successful DTLS handshake only provides confidentiality and integrity. Authenticity
+is achieved by succeeding a [Noise handshake](../secure-comm/noise) following
+the DTLS handshake.
+
+<!-- TO ADD DIAGRAM -->
+
+### Coming soon: Browser-to-Browser
+
+Eventually, libp2p will have support for communication between two
+browsers.
+
+The technical specification and initial implementations of WebRTC
+Browser-to-Browser connectivity is planned for release in early 2023.
+Track the progress [here](https://github.com/libp2p/specs/issues/475).
+
+<!-- TO ADD DIAGRAM -->
+
+{{< alert icon="💡" context="note" text="See the WebRTC <a class=\"text-muted\" href=\"https://github.com/libp2p/specs/blob/master/webrtc/README.md\">technical specification</a> for more details." />}}
+
+## Comparing WebRTC and WebTransport
+
+In general, WebTransport is a lower-level technology that provides a
+standard interface for delivering data, whereas WebRTC is a higher-level technology
+that offers a set of APIs and protocols for building communication applications.
```

Comment:
I am not sure I agree. Both cover the same parts in a networking stack. Both provide encryption. Both provide multiplexing. On the surface users interact with both on the multiplexing layer. What makes one more low-level than the other?

Reply:
Right, I suppose the idea I'm trying to convey here is that WebTransport doesn't not provide functionality for building communication applications, while WebRTC does.

Reply:
Yes and no. This is correct when you look at audio and video streaming (MoQ will build on top of WebTransport to enable that). However, for us - using only WebRTC data channels - they do cover the same part of the networking stack.

I suggest rewording this. You could say that WebRTC was primarily built for in-browser audio and video communication, whereas WebTransport aims to offer a general-purpose bidirectional byte-stream interface between a browser and a server.